packages/tests/esm-misc/package.json

```diff
@@ -35,8 +35,5 @@
     "ts-node": "^10.9.2",
     "tsconfig-paths": "^4.2.0",
     "typescript": "5.9.2"
-  },
-  "imports": {
-    "src/": "src/*"
```

Comment:
Node subpath imports must start with `#`, so this configuration was invalid. I'm working on subpath import support in another branch and will likely create a new integration test for that, specifically.